src/types.ts

```diff
@@ -8,11 +8,11 @@ export interface PluginConfig extends Config {
      */
     readonly accessKeyName?: string
     /**
-     * Bucket name
+     * Bucket configuration
      *
      * @default ""
      */
-    readonly bucketName: string
```

Comment:
Shouldn't this be called something else since it's not just a bucket name anymore?